openshift-ci/build-root/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Dockerfile to bootstrap build and test in openshift-ci
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
+RUN yum -y install dnf httpd-tools
+
+RUN dnf -y install 'dnf-command(config-manager)'
+
+RUN dnf -y config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.repo
+
+RUN dnf -y install gh
```

Comment:
For now remove line no 7 to 11. Later we will add dependencies as per the need

Reply:
Done with b09222c8ba25e1c5d1cdc88a3b890335f070662a